frontend/src/user/contest/pages/index.vue

```diff
@@ -5,96 +5,36 @@ import IconAnglesRight from '~icons/fa6-solid/angles-right'
 import IconCaretDown from '~icons/fa6-solid/caret-down'
 import IconCaretUp from '~icons/fa6-solid/caret-up'
 import { useTimeAgo } from '@vueuse/core'
-
+import { onMounted } from 'vue'
+import axios from 'axios'
 import { ref } from 'vue'
```

Comment:
사소하긴 한데 import 정리 부탁드려요!

```suggestion
import { ref, onMounted } from 'vue'
```

Reply:
@

> 사소하긴 한데 import 정리 부탁드려요!

수정 완료했습니다~